package.json

```diff
@@ -31,7 +31,8 @@
     "redux-thunk": "~2.3.0",
     "shortid": "^2.2.15",
     "typescript": "~3.8.2",
-    "uuid": "^7.0.1"
+    "uuid": "^7.0.1",
+    "yarn": "~1.22.4"
```

Comment:
You are right, this can be removed. Yarn is expected to be globally installed.

Reply:
Ok, I'll take that out in the next commit.